### ISSUE TITLE
Cannot Host Servers Anymore

### ISSUE THREAD

**Issue by Teivellus** [Regalis11/Barotrauma#3508]
[*] I have searched the issue tracker to check if the issue has already been reported.

**Description**
For a while, now I have been unable to host servers. Missions and sandbox just die straight away and campaign lets me load in but if I have to load an outpost in the next map, it crashes. So, the problem is when I want to leave the station you start on, start a mission, or start sandbox; I get the following message:

---

Generating an outpost for the end of the level... (Location; Milda City, level type: LocationConnection.) 

Error while starting the round (did not receive STARTGAMEFINALIZE message from the server). Stopping the round...

TimedOut

Coroutine "Barotrauma.Networking.GameClient+d__82" has failed.

---

Afterwards, the round crashes... I can see that the code for STARTGAMEFINALIZE is in the repository for Bartotrauma on this website, but I can't do anything about this bug.

**Steps To Reproduce**
No Idea

**Version**
Current version. Windows 10 64 bit.

**Comment by mygamingaccount** [comment]
Did this happen during the steam maintenance downtime? Does it work now that it's over?

**Comment by Teivellus** [comment]
> Did this happen during the steam maintenance downtime? Does it work now that it's over?

This was for a few days.

I verified game files, and that didn't work.
I uninstalled Barotrauma, reinstalled it, and that didn't work.
I then uninstalled again, deleted every single file in the barotrauma folders after backing up my mods and submarines. Reinstalled, pasted my subs and mods back in.

And you know what? Now it works. Some file in that damn folder must have been broken.

If anyone gets that same problem, there's the fix.
